quickstarts/elixir.mdx

```diff
@@ -0,0 +1,348 @@
+---
+title: 'Elixir'
+description: 'Learn how to use Novu to quickly send multi-channel (SMS, Email, Chat, Push) notifications with Elixir.'
+---
+Let's get into it! 🚀
+
+# Requirements
+
+To be able to use this Quickstart, you would need to have the following pre-requisite:
+
+- A Novu account. If you are new, [sign up](https://web.novu.co/) for free.
+- A working Elixir project.
+
+# Install and Set up the Novu Elixir SDK in your project
+
+Novu is available on Hex Package Manager - [hex.pm](https://hex.pm/packages/novu). 
+
+1. Add Dependency in mix.exs:
+Open your project's 'mix.exs' file and add the dependency to the 'deps' function.
+
+```exs
+
+def deps do
+  [
+    {:novu, "~> 0.2.0"}  #replace "~> 0.2.0" with the appropriate version you want to use.
+  ]
+end
+```
+
+The latest version can be found [on GitHub.](https://github.com/novuhq/novu-elixir#installation)
+
+2. Sync Your Project:
+After adding the dependency, you can run the following command in your Elixir project's root directory to fetch and compile the new dependency:
+
+```elixir
+mix deps.get
+```
+
+3. Initialize the Library:
+To initialize the library and configure it with your API key, you can create an instance of the Novu module in one of your Elixir modules or scripts.
+
+```elixir
```

Comment:
This code block is incorrect. To initialize the library, you set the api key and domain in your application configuration.

https://github.com/novuhq/novu-elixir/blob/main/lib/novu/http.ex#L78

In your `config/config.exs` file:

```elixir
import Config

config :novu,
  api_key: "API_KEY",
  domain: "DOMAIN"
```